### ISSUE TITLE
Error when migrating crono migration

### ISSUE THREAD

**Issue by ferdi2005** [plashchynski/crono#82]
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateCronoJobs < ActiveRecord::Migration[4.2]

**Comment by alliedarmour** [comment]
Just replace the [4.2] with the version you are using, in my case it's [6.0]

**Comment by ClikeX** [comment]
The issue here wasn't the [4.2]. It's the fact that Crono's generated migration doesn't have the version tag in it at all.

My migration came like this

```ruby
class CreateCronoJobs < ActiveRecord::Migration
  def self.up
    create_table :crono_jobs do |t|
      t.string :job_id, null: false
      t.text :log, limit: 1073741823 # LONGTEXT for MySQL
      t.datetime :last_performed_at
      t.boolean :healthy
      t.timestamps null: false
    end
    add_index :crono_jobs, [:job_id], unique: true
  end

  def self.down
    drop_table :crono_jobs
  end
end
```

**Comment by plashchynski** [comment]
Fixed in the new version of the gem (2.0.0)
